dist/systemd/obsapidelayed-clockworkd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description = OBS API Delayed clockworkd server
+BindsTo = obsapidelayed.target
+
+[Service]
+Environment = "RAILS_ENV=production"
+User = wwwrun
+Group = www
+WorkingDirectory = /srv/www/obs/api
+ExecStart = /usr/bin/bundle.ruby2.5 exec /usr/bin/clockworkd.ruby2.5 --log-dir=log -l -c config/clock.rb start
```

Comment:
As far as I remember this path is not the right one anymore. See [dist/obsapidelayed](https://github.com/openSUSE/open-build-service/blob/master/dist/obsapidelayed#L41)...

Reply:
@hennevogel 

> As far as I remember this path is not the right one anymore. See dist/obsapidelayed...

Changed.